graylog2-server/src/main/java/org/graylog2/indexer/ranges/IndexRangeService.java

```diff
@@ -31,4 +31,5 @@
     void save(IndexRange indexRange);
 
     IndexRange calculateRange(String index);
+    IndexRange createForDeflector(String index);
```

Comment:
I think `createEmpty` would be a better name as it's not inherently bound to the deflector alias (it's not even being used in the method implementations).

Maybe adding it as a default method (https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html) would make sense here as its implementation doesn't rely on any internals of the specific implementation of `IndexRangeService`.